src/grisu3.c

```diff
@@ -0,0 +1,361 @@
+/* This file is part of an implementation of the "grisu3" double to string
```

Comment:
Can you include the license too? Might need to copy and paste from somewhere else